### ISSUE TITLE
@html-eslint/indent rule breaks for Windows line endings crlf

### ISSUE THREAD

**Issue by reinholdk** [yeonjuan/html-eslint#85]
Running @html-eslint on Windows is not considering Windows line endings and failing therefore:

Config:
`'@html-eslint/indent': ['error', 2],`

File:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Test</title>
  </head>
  <body>
    <span>
      <i>some</i>
      thing
    </span>
  </body>
</html>
```

Error:
results in `8:19  error  Expected indentation of 6 space but found 5 space  @html-eslint/indent`

**Comment by yeonjuan** [comment]
@reinholdk Thanks for the reporting. I'll check it

**Comment by reinholdk** [comment]
@yeonjuan thanks, was briefly looking into the rule to see whether I could propose a quick fix. However, I feel the parser needs to be changed as well.

**Comment by yeonjuan** [comment]
@reinholdk You are right. :) 
Current html-eslint/parser didn't give tokens to eslint core. It should be fixed but it would be not easy and takes long time. (Anyway I'll do it someday)
So, in now, it would be really appreciated if you gives a quick fix propose.

**Comment by reinholdk** [comment]
@yeonjuan tried fixing it here: #86

**Comment by yeonjuan** [comment]
@reinholdk Thanks for the PR! I left a suggestion for it. I'm now working on develop new parser for html-eslint. your proposal gave me a nice case about new parser should handled. Great thanks!
